lnwallet/mock.go

```diff
@@ -364,3 +364,9 @@ func (*mockChainIO) GetBlock(blockHash *chainhash.Hash) (*wire.MsgBlock,
 
 	return nil, nil
 }
+
+func (*mockChainIO) GetBlockHeader(blockHash *chainhash.Hash) (*wire.BlockHeader,
```

Comment:
Should be in the same commit that the interfaces is expanded so the commit builds.